.github/ISSUE_TEMPLATE/update_deps.md

```diff
@@ -0,0 +1,46 @@
+---
+name: Update critical dependencies between SDK and Canton
+about: This template outlines the workflow for updating critical dependencies that must be synchronized between the SDK and Canton repositories. Examples include gRPC, Pekko, OpenTelemetry, etc.
+title: "Update ..."
+
+---
+
+<!-- list the dependencies to be updated here -->
+
```

Comment:
```suggestion

Note: Since the process involves synchronous changes to both the SDK and Canton repositories, it may conflict with the daily SDK update task. To avoid issues, please coordinate with the person responsible for the SDK update via the #team-sdk-update Slack channel."

```